Automattic/newspack-popups
# Add semantic-release & linting

### All Submissions:






### Changes proposed in this Pull Request:



- Configures semantic-release for automated releases
- Configures JS & CSS liniting
- Configures CircleCI to run the linting and releasing

### How to test the changes in this Pull Request:

1. `npm run lint` should exit without error
2. `npm run format:js` & `npm run format:scss` should not change any files

### Other information:
